docs/usage/CATEGORY_SYNC.md

```diff
@@ -115,7 +85,17 @@ __Note__ The statistics object contains the processing time of the last batch on
 More examples of how to use the sync 
 1. From another CTP project as a source can be found [here](/src/integration-test/java/com/commercetools/sync/integration/ctpprojectsource/categories/CategorySyncIT.java).
 2. From an external source can be found [here](/src/integration-test/java/com/commercetools/sync/integration/externalsource/categories/CategorySyncIT.java). 
+
+##### Usage Tip
+The sync library is not meant to be executed in a parallel fashion. Check the example in [point #2 here](/docs/usage/PRODUCT_SYNC.md#caveats). 
+By design, scaling the sync process should **not** be done by executing the batches themselves in parallel. However, it can be done either by:
+  - Changing the number of [max parallel requests](/src/main/java/com/commercetools/sync/commons/utils/ClientConfigurationUtils.java#L116) within the `sphereClient` configuration. It defines how many requests the client can execute in parallel.
+  - or changing the draft [batch size](https://commercetools.github.io/commercetools-sync-java/v/v1.0.0-M14/com/commercetools/sync/commons/BaseSyncOptionsBuilder.html#batchSize-int-). It defines how many drafts can one batch contain.
  
+The current overridable default [configuration](/src/main/java/com/commercetools/sync/commons/utils/ClientConfigurationUtils.java#L45) of the `sphereClient` 
+is the recommended good balance for stability and performance for the sync process.
+
+In order to exploit the number of `max parallel requests`, the `batch size` should have a value set which is equal or higher.
```

Comment:
```suggestion
In order to exploit the number of `max parallel requests`, the `batch size` should have a value set, which is equal or higher.
```

Reply:
I don't this comma is grammatically correct: https://en.oxforddictionaries.com/punctuation/comma